pkg/operator/ceph/csi/spec.go

```diff
@@ -547,7 +605,10 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		}
 		if multusApplied {
 			cephfsPlugin.Spec.Template.Spec.HostNetwork = false
```

Comment:
To be clear, HostNetwork can be false in all cases with the new csi driver using nsenter, but to lower risk we just aren't going to change that yet?

Reply:
Yes, we can change it later once this looks stale, for now, csi will use pod networking only for multus.